packages/material-ui/src/Backdrop/Backdrop.test.js

```diff
@@ -1,27 +1,27 @@
 import * as React from 'react';
 import { expect } from 'chai';
-import { getClasses, createMount, createClientRender, describeConformance } from 'test/utils';
+import { createMount, createClientRender, describeConformanceV5 } from 'test/utils';
 import Backdrop from './Backdrop';
 import Fade from '../Fade';
+import classes from './backdropClasses';
 
 describe('<Backdrop />', () => {
   const mount = createMount({ strict: true });
   const render = createClientRender();
-  let classes;
 
-  before(() => {
-    classes = getClasses(<Backdrop open />);
-  });
-
-  describeConformance(<Backdrop open />, () => ({
+  describeConformanceV5(<Backdrop open />, () => ({
     classes,
     inheritComponent: Fade,
     mount,
     refInstanceof: window.HTMLDivElement,
+    muiName: 'MuiBackdrop',
     skip: [
       'componentProp',
+      'componentsProp',
       // react-transition-group issue
       'reactTestRenderer',
+      'themeVariants',
```

Comment:
We shouldtest the `themeVariants` and the `themeDefaultProps`, no reason to skip them. We can use the `invisible` prop for testing the `themeVariants`.